astrokat/test/check_nd_units.sh

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+
+python -m unittest astrokat.test.test_nd_timings.TestAstrokatYAML.test_nd_pattern_sim
+python -m unittest astrokat.test.test_nd_timings.TestAstrokatYAML.test_nd_pattern_ants
+python -m unittest astrokat.test.test_nd_timings.TestAstrokatYAML.test_nd_pattern_plus_off
+python -m unittest astrokat.test.test_nd_timings.TestAstrokatYAML.test_nd_trigger_long
+python -m unittest astrokat.test.test_nd_timings.TestAstrokatYAML.test_nd_trigger_short
```

Comment:
These scripts may be useful, but are a bit of extra maintenance.  E.g., when we add more tests.

Reply:
No, wrong branch, these bash scripts are for me -- I have a devel branch that I stick this type of stuff in
Sorry this was sloppy, I cleaned up the tests